### ISSUE TITLE
reprocess datasets through MAG workflow

### ISSUE THREAD

**Issue by aclum** [microbiomedata/issues#619]
**RACI**

_Tag people in their roles_
- Responsible: @Michal-Babins 
- Accountable: @hubin-keio 
- Consulted: @chienchi 
- Informed: @aclum 

**Describe the the task**
- [ ] reprocess workflow runs since July 2023 (NEON, TRiP) due to error in binning workflow which specified incorrect path to checkM reference data

**Criteria for completion**
- new workflow results with .2 identifiers should be in mongo prod with a processed data greater than Feb 20, 2024. 

**Estimate people time**
- [Hours or days of people time. 1 person, 4 hours]

**Completion Date (Goal)**
- [GOAL!]

**Target Sprint Start & End Dates**
- Start:
- End:

**Tag Blocker/Contingent upon issues**
-  @chienchi  to confirm end to end workflow test completes for a dataset where we expect bins to be generated.

**Comment by aclum** [comment]
related to https://github.com/microbiomedata/metaMAGs/issues/19

**Comment by Michal-Babins** [comment]
I am gathering the neccessary information to reprocess these. I will have @scanon review this. Once the mags chances are ready, we can resubmit trip and neon soil for mags only with the bumped version. We can start with TriP first since it's smaller and move to NEON.

**Comment by Michal-Babins** [comment]
I am running a few more tests to make sure the runtime attributes are set correctly with the updates, the changes @chienchi made look good, but need to confirm that the runtime requirements are stable. I had one project crash on me due to OOM and  the requested memory was too low.

**Comment by aclum** [comment]
We need to add the new file types including capturing the low quality bins, see https://github.com/microbiomedata/nmdc-schema/pull/1791. I'm working on getting runtime dev updated.

**Comment by ssarrafan** [comment]
@Michal-Babins @aclum what is the status of this issue?

**Comment by aclum** [comment]
Michal was testing the runtime requirements and we need an updated runtime prod env to be able to handle  the new file type numerations. This first needs a new update to runtime dev. We updated runtime dev to 10.1 but that release is incomplete per Patrick C. so we need runtime dev to be updated to 10.1.1, then test, then runtime prod needs to be updated to 10.1.1. cc @kaijli

**Comment by ssarrafan** [comment]
> Michal was testing the runtime requirements and we need an updated runtime prod env to be able to handle the new file type numerations. This first needs a new update to runtime dev. We updated runtime dev to 10.1 but that release is incomplete per Patrick C. so we need runtime dev to be updated to 10.1.1, then test, then runtime prod needs to be updated to 10.1.1. cc @kaijli

Thanks Alicia.  Should this be moved to the next sprint?

**Comment by ssarrafan** [comment]
Removing from sprint.  @aclum @kaijli let me know if this is still active.

**Comment by aclum** [comment]
We need to resolve the blockers first, adding backlog label so we don't lose this.

**Comment by aclum** [comment]
@aclum  to add study IDs.

**Comment by aclum** [comment]
[nmdc.mags_activity_set.csv](https://github.com/microbiomedata/issues/files/14594604/nmdc.mags_activity_set.csv)

**Comment by aclum** [comment]
Reattaching the list that I provided in slack on July 28th 2023. These are projects where we know the JGI workflow produced bins with a mapping to NMDC's corresponding omics ID.

[IMG (15).xlsx](https://github.com/microbiomedata/issues/files/14594634/IMG.15.xlsx)

**Comment by aclum** [comment]
Moving this to blocked as this is on hold until we get the eukcc component code from Neha.

**Comment by aclum** [comment]
Shane is testing some minor changes to remove extra paths for the low quality bins, after that we can start processing the backlog.

**Comment by ssarrafan** [comment]
@aclum @scanon @kaijli are we still waiting for Shane's testing to be done?

**Comment by aclum** [comment]
Yes, the last test we did did not have one of the necessary files. I have a [draft PR](https://github.com/microbiomedata/nmdc_automation/pull/216) which i believe addresses that issue but I need input from Shane.

**Comment by aclum** [comment]
Moving this to the next sprint since the PR is still not merged.

**Comment by aclum** [comment]
This is blocked on speed improvements Shane is making. Those are in testing. Moving this to the next sprint.

**Comment by aclum** [comment]
blocked on testing of version MAGs 1.3.10, hopefully we can start running these this sprint.

**Comment by aclum** [comment]
We have a working version of the binning pipeline, versions with issues with the checkM database path have been failed out of the system and the individual study tickets is being used to track if binning is complete so closing this ticket.
